### DISCUSSION TITLE
Will Nuxt 2 continue to get updates?

### DISCUSSION THREAD

**Discussion by gplusdotgr** [nuxt/nuxt#10452] [Questions (Nuxt 2)] [answered]
The last release was in August 2021, will there be more or shall we already consider migrating?

**Comment by TheAlexLichter** [comment, marked as answer]
As described on the [roadmap](https://v3.nuxtjs.org/community/roadmap/#-release-schedule), there is a `v2.16` release planned containing "Nuxt v2 cumulative updates for future compatibility". Nuxt 2.x is in `maintenance` mode though.

You can consider migrating your project to Nuxt Bridge (or directly to Nuxt 3) if you like. But keep in mind that Nuxt is in the **release candidate** stage at the time of writing ☺️ 

There is nothing wrong staying with Nuxt2 at the moment though. Things will still work as they did before 😋

**Comment by nayils** [comment]
Will nuxt bridge continue to be supported as a continuation of nuxt 2? Or will it close eventually?

Does it make sense to start a project on nuxt bridge? Nuxt 3 doesn't support my plugins yet.
